src/contactCard/Summary.tsx

```diff
@@ -18,10 +18,19 @@ export function renderSummary(
             {renderContactSummary(profile, onContactDetailsClick)}
             {renderOrgSummary(manager, isManagerLoading, onOrgDetailsClick, onPersonaClick)}
         </ul>
-    );
-}
-
+    ); 
+    }
 
+    let isExpanded = false;
+    function handleContactDetailsClick(onContactDetailsClick: () => void) {
+        onContactDetailsClick();
+        const button = document.querySelector('[data-focus="button"]');
```

Comment:
This may lead to bugs if multiple contact cards are shown on the screen. Since all of them will have data-focus button, the query select will select all of them.